src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagementGetTasksBySearchCriteriaConsumerTest.java

```diff
@@ -580,6 +611,22 @@ private String createSearchWithTaskTypeRequest() {
             """;
     }
 
+    private String createSearchWithTaskStateRequest() {
```

Comment:
Naming can be better since this is specifically requesting COMPLETED Tasks only.